src/main.rs

```diff
@@ -46,8 +46,6 @@ fn parse_line(s: &String) -> Result<String, ()> {
 }
 
 fn main() {
-    let cli = Cli::parse();
```

Comment:
can you just make this 

`let _cli  = Cli::parse(); `

Otherwise your dropping the help information.

Reply:
Made the change